packages/nx/src/command-line/graph/graph.ts

```diff
@@ -423,7 +423,7 @@ async function startServer(
       return;
     }
 
-    let pathname = join(__dirname, '../../core/graph/', sanitizePath);
+    let pathname = join(__dirname, '../core/graph/', sanitizePath);
```

Comment:
These paths seem incorrect now. This would resolve to packages/nx/src/command-line/core, instead of packages/nx/src/core right?